guides/app-guides/borrow-ageur.md

```diff
@@ -0,0 +1,79 @@
+---
+description: How to borrow agEUR from the Angle App
+---
+
+# Borrowing agEUR 
+
+The [Angle Borrowing module](/borrowing-module/README.md) introduced new features to Angle and agEUR. You can now borrow agEUR or get leverage on your crypto directly from the [Angle App](https://app.angle.money/#/borrow). 
```

Comment:
Keep the guides intemporal -> The Angle Borrowing module lets you borrow agEUR from the Angle App